### PR TITLE
fix: relaunch mekong

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: kubernetes/devnet-0/assertoor/values.yaml [hunk 1]
```diff
@@ -399,7 +399,7 @@ assertoor:
       executionUrl: https://<path:/secrets/services/services.enc.yaml#ethereum | jsonPath {.testnets.mekong-devnets.node_ingress.combined}>@rpc.teku-reth-2.mekong-devnet-0.ethpandaops.io
       consensusUrl: https://<path:/secrets/services/services.enc.yaml#ethereum | jsonPath {.testnets.mekong-devnets.node_ingress.combined}>@bn.teku-reth-2.mekong-devnet-0.ethpandaops.io
 
-  validatorNamesInventory: https://config.mekong.ethpandaops.io/api/v1/nodes/validator-ranges
+  validatorNamesInventory: https://config.mekong-devnet-0.ethpandaops.io/api/v1/nodes/validator-ranges
```
Comment (pk910) [inline kubernetes/devnet-0/assertoor/values.yaml:402]: intended change to long url?

File: kubernetes/devnet-0/homepage/values.yaml [hunk 2]
```diff
@@ -100,6 +100,7 @@ testnet-homepage:
         doraUrl: https://dora.mekong.ethpandaops.io
         checkpointSyncUrl: "https://checkpoint-sync.mekong.ethpandaops.io"
         blobscanUrl: https://blobscan.com
+        blobscanethpandaUrl: https://blobscan.mekong.ethpandaops.io
```
Comment (pk910) [inline kubernetes/devnet-0/homepage/values.yaml:103]: intended addition of blobscan?

**Review by pk910 — APPROVED** [review]

File: kubernetes/devnet-0/homepage/values.yaml (hunk 2 above)
Comment (barnabasbusa) [reply kubernetes/devnet-0/homepage/values.yaml:103]: its still commented out below.

File: kubernetes/devnet-0/assertoor/values.yaml (hunk 1 above)
Comment (barnabasbusa) [reply kubernetes/devnet-0/assertoor/values.yaml:402]: yeah think it wasn't working before :D

**Pull request merged**